.gitattributes

```diff
@@ -0,0 +1,3 @@
+/.* export-ignore
+/tests export-ignore
+/phpunit.xml.dist export-ignore
```

Comment:
The lines in `.gitattributes` are not regular expressions, they are normal *nix patterns: the dot does not mean `any char`, simply means `.`

You can see the result of this `.gitattributes` files by downloading the archive from the branch originating this pull request: https://github.com/mlocati/php-punycode/archive/patch-1.zip

As you can see, in the root directory we still have these items:
- `/src/`
- `/CHANGELOG.md`
- `/composer.json`
- `/LICENSE`
- `/README.md`